herbert_core/classes/MPIFramework/@JobExecutor/JobExecutor.m

```diff
@@ -20,6 +21,7 @@
     % JobExecutor Properties:
     %
     % labIndex          - The number of the running task.
+    % numLabs           - The number of running tasks.
```

Comment:
Just to say it could be good to say why "lab" is used here. Presuming it means label? taskIndex etc look more intuitive

Reply:
used Matlab convention for openMPI interface, which calls independent MPI worker(s) -- lab(s)